src/components/LocaleChooser.vue

```diff
@@ -62,16 +62,16 @@ export default {
 }
 .locale-chooser-field .control .select select {
     padding-left: 2.5em;
-    z-index: -1;
 }
 .locale-chooser-field .control .select::before {
     content: url("../assets/language.png");
     position: absolute;
     left:10px;
     top:8px;
+    z-index: 1;
 }
 .locale-chooser .select:not(.is-multiple):not(.is-loading)::after {
-     z-index: 0;
+     z-index: 1;
```

Comment:
I've reworked positioning for locale chooser in another PR, so you can leave it as it was before.
I just noticed that github shows me that I haven't reviewed your PR, although I did leave a reply. It doesn't let me click 'request changes' if I don't add any reply now. So, I added this note to be able to click 'request changes' :)

Reply:
Ok great will try to clear off the remaining issue just now.